components/ide/jetbrains/backend-plugin/src/main/kotlin/io/gitpod/jetbrains/remote/actions/AccessControlAction.kt

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+// Licensed under the GNU Affero General Public License (AGPL).
+// See License-AGPL.txt in the project root for license information.
+
+package io.gitpod.jetbrains.remote.actions
+
+import com.intellij.openapi.actionSystem.AnAction
+import com.intellij.openapi.actionSystem.AnActionEvent
+import com.intellij.openapi.components.service
+import io.gitpod.jetbrains.remote.GitpodManager
+import org.apache.http.client.utils.URIBuilder
+
+class AccessControlAction : AnAction() {
+    private val manager = service<GitpodManager>()
+
+    override fun actionPerformed(event: AnActionEvent) {
+        manager.pendingInfo.thenAccept { workspaceInfo ->
+            URIBuilder(workspaceInfo.gitpodHost).setPath("integrations").build().toString().let { url ->
```

Comment:
In [VS Code](https://github.com/gitpod-io/openvscode-server/blob/gp-code/main/extensions/gitpod-shared/src/features.ts#L353) the paths are retrieved from [gitpod-protocol](https://github.com/gitpod-io/gitpod/blob/main/components/gitpod-protocol/src/util/gitpod-host-url.ts#L108-L110), I would prefer if we could extend the Java binding instead of hardcoding the paths here, it will be painful to remember if they change. 

Did you consider this approach? 🙏

Reply:
> I am curious to know what do you think are the challenges in going with that approach first?

It's because the I'll be away next week :P So the follow-up PR won't be coming soon. I thought it would be better to have those links already there in this first PR, and replaced in the follow-up (which in turn would be focused in bring the server APIs).

Reply:
Ah, I see. I am not against merging as it is, as long as we allocate time next week. I am happy to work on the follow-up PR.

Reply:
Thanks, Andrea! I'd appreciate if you could take over this PR/branch and complete it during next week (besides working on the follow-up PR). I'll assign this PR to you too.